ISHPermissionKit/ISHPermissionsViewController.m

```diff
@@ -191,6 +191,10 @@ - (void)permissionRequestViewController:(ISHPermissionRequestViewController *)vc
         ISHPermissionRequest *req = [self requestAtIndex:nextIndex];
         nextState = [req permissionState];
     } while (nextIndex < self.permissionCategories.count && !ISHPermissionStateAllowsUserPrompt(nextState));
+
+    if (error && self.errorBlock) {
+        self.errorBlock(vc.permissionCategory, (NSError * _Nonnull)error);
```

Comment:
Could this cast go away if we removed the `nullable` annotation in line 177? Since we don't use `nullability` in .m consistently, we might as well remove it there.